sdks/csharp/test/csharp-sdk-test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>

```

Comment:
NET Core 2.2 support ended in December 2019 so hopefully this is fine to update to the previous NET Core LTS (current LTS is NET 6). Let me know if you'd rather not update. This is only the test project, the SDK library itself is still targeting NET Standard 2.0.